articles/industrial-iot/industrial-iot-platform-versions.md

```diff
@@ -5,21 +5,25 @@ author: jehona-m
 ms.author: jemorina
 ms.service: industrial-iot
 ms.topic: overview
-ms.date: 03/08/2021
+ms.date: 03/11/2021
 ---
-# Azure Industrial IoT Platform v2.8.0 LTS
+# Azure Industrial IoT Platform Patch Release v2.8.1 now available
```

Comment:
* Let's remove `v` from `v2.8.1`.
* I would suggest to have information about `2.8.1` version after `2.8.0`.

Reply:
v is now removed from the version to align with our GitHub docs.
For second point, I would say generally latest releases should come on top for better readability about latest versions.

Reply:
OK, sounds good.